pkgs/native_toolchain_c/test/clinker/testfiles/linker/test2.c

```diff
@@ -0,0 +1,6 @@
+#include <stdio.h>
```

Comment:
Can these files live in test_data instead? (to keep it consistent with where other test data lives in other packages)